docs/en/observability/monitor-nginx.asciidoc

```diff
@@ -209,7 +210,8 @@ After your nginx logs are ingested, view and explore your logs using <<monitor-n
 ==== Logs Explorer
 
 With Logs Explorer, you can quickly search and filter your log data, get information about the structure of log fields, and display your findings in a visualization.
-To view nginx logs, open Kibana and go to *{observability}* → *Logs Explorer*.
+
+To open **Logs Explorer**, find **Discover** in the main menu or use the {kibana-ref}/introduction.html#kibana-navigation-search[global search field].
```

Comment:
The new guidelines don't work here because the Logs Explorer is under Discover in the new nav, but in the old nav it's under Logs. 
![image](https://github.com/user-attachments/assets/5bb95314-b615-4c18-9707-33014e017d92)

Reply:
Decision was to recommend the global search bar in this situation.